.env.example

```diff
@@ -40,6 +40,7 @@ ZOOKEEPER_IMAGE_TAG=3.8.1
 APACHE_DRUID_IMAGE_TAG=24.0.2-rc1
 BLACKFIRE_IMAGE_TAG=2.16.1
 JENKINS_IMAGE_TAG=2.409
+MONGO_IMAGE_TAG=jammy
```

Comment:
Please use specific image tag. Example 1.2.3, 2.3.4-beta etc.

Reply:
Mongo Image Tag is updated.